src/java.base/share/conf/security/java.security

```diff
@@ -28,6 +28,46 @@
 # Properties in this file are typically parsed only once. If any of the
 # properties are modified, applications should be restarted to ensure the
 # changes are properly reflected.
+#
+# The special "include" property can be defined one or multiple times in
+# this file with a filesystem path value. The effect of each definition
+# is to include a referred security properties file inline, adding all
+# its properties. Security properties defined before an include statement
+# may be overridden by properties in the included file, if their names
+# match. Conversely, properties defined after an include statement may
+# override properties in the included file.
+#
+# Included files, as well as files pointed by java.security.properties,
+# can include other files recursively. Paths may be absolute or relative.
+# Each relative path is resolved against the base file containing its
+# "include" definition, if local. Paths may contain system properties for
+# expansion in the form of ${system.property}. If a system property does
+# not have a value, it expands to the empty string.
+#
+# System properties in included paths provide flexibility for users to
+# switch between predefined security profiles on a per-execution basis.
+# Profile selection may be required or optional depending on the files
+# layout, as shown in example #3 below.
```

Comment:
I feel like this paragraph is better suited to a programmer's guide. I think the text for this property is already quite long, and we should focus mostly on specification and behavior and leave text that sounds like advice or use cases to a programmer's guide, or release note.

Reply:
Yes, we absolutely agree. This is exactly the reason why we didn't introduce this discussion about profiles and possible file layouts in the _java.security_ file in the first place. As we see it, the documentation in the _java.security_ file should be more concise and focused on syntax and behavior.